productivity-suite/app/legacy-app/src/components/layout.css

```diff
@@ -1,206 +1,221 @@
 .app-header {
-    display: flex;
```

Comment:
are most of the changes in this file caused by auto-formatting? We haven't actually configured prettier in this demo/repo, I guess we should? what do you think @petebacondarwin?

Reply:
yeah, they are. the significant changes were to the footer to make it sticky.

yes +1 to prettier or disregard formatting completely in review.